test/adaptive_load/fake_plugins/fake_metrics_plugin/fake_metrics_plugin.cc

```diff
@@ -41,6 +41,18 @@ absl::StatusOr<double> FakeMetricsPlugin::GetMetricByName(absl::string_view metr
   return value_or_error_from_name_[metric_name];
 }
 
+FakeMetricsPluginWithMeasuringPeriod::FakeMetricsPluginWithMeasuringPeriod(
+    const nighthawk::adaptive_load::FakeMetricsPluginConfig& config)
+    : FakeMetricsPlugin(config) {}
+
+absl::StatusOr<double> FakeMetricsPluginWithMeasuringPeriod::GetMetricByNameWithMeasuringPeriod(
```

Comment:
Does this return `absl::InternalError` unconditionally?

If yes, how are we testing the case where a plugin has the `WithMeasuringPeriod` method implemented and returns value?

Reply:
Opted for a Mock approach instead.